Jenkinsfile

```diff
@@ -96,7 +96,7 @@ pipeline {
 		maven 'apache-maven-latest'
 	}
 	environment {
-		MAVEN_OPTS = "-Xmx4G"
+		MAVEN_OPTS = "-Xmx3G"
```

Comment:
Still a very big number on a node limited to 4g.

Reply:
That's right.
I'll experiment with lower numbers.

In general I think its is better to be at the lower limit than at the upper, since the GC can maybe try hard to free heap memory. And OOMErrors of the JVM can be analyzed through a heap-dump more easy (to find possible memory 'leaks'), which is probably impossible/harder if the executor/step gets killed due to excessive memory demand.